src/main/java/net/dv8tion/jda/internal/entities/UserImpl.java

```diff
@@ -216,6 +231,13 @@ public UserImpl setFake(boolean fake)
         this.fake = fake;
         return this;
     }
+    
+    public UserImpl setFlags(int flags)
+    {
+        this.flagsRaw = flags;
+        this.flagsSet = UserFlag.getFlags(flags);
```

Comment:
Always converting these to an EnumSet is a bad idea, most user instances won't need this representation.

Reply:
removed the flagSet